tests/unit/hal/test_tei.py

```diff
@@ -0,0 +1,216 @@
+"""Unit tests for the TEI exporter."""
+
+import mock
+import pytest
+
+from inspirehep.modules.hal import tei
+from invenio_records.api import Record
```

Comment:
`invenio_records` import should be above, in its own block of imports:
```python
import mock
import pytest

from invenio_records.api import Record

from inspirehep.modules.hal import tei
```
The general idea is: future stdlib vendored invenio inspire relative